src/divoom.js

```diff
@@ -63,3 +63,19 @@ module.exports.setDateTime = async date => {
     for (const buffer of d.messages.asBinaryBuffer())
         await write(buffer);
 };
+
+
+/**
+ * Display Image on the Divoom device
```

Comment:
```suggestion
 * Display an image on the Divoom device.
```
Consistent wording with other functions